### PR TITLE
Replace formatUser with UserModel.name()

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
@msmolens PTAL.

At some point, we should start caching the models created by owner model accessor methods, but I don't see any performance issues at the moment.

File: web_external/js/views/body/StudyResultsView.js [hunk 1]
```diff
@@ -589,10 +587,7 @@ isic.views.StudyResultsView = isic.View.extend({
             this.images.reset(imageModels);
 
             // Populate users collection
-            var userModels = _.map(this.study.get('users'), function (user) {
-                return new girder.models.UserModel(user);
-            });
-            this.users.reset(userModels);
+            this.users.reset(this.study.users());
```
Comment (msmolens) [inline web_external/js/views/body/StudyResultsView.js]: `reset()` can't handle a `Collection`, but we could use `this.study.users().models`.

File: web_external/templates/body/studyPage.jade [hunk 2]
```diff
@@ -10,8 +10,8 @@
   .isic-listing-content.isic-study-users-content
     if study.get('users').length
       ul
-        each user in study.get('users')
-          li= formatUser(user)
+        each user in study.users().toArray()
```
Comment (msmolens) [inline web_external/templates/body/studyPage.jade]: `study.users().models` gives raw access to the collection's model array, which would be more efficient.

**Comment by brianhelba** [comment]
@msmolens Fixed both places.

**Pull request merged**